glad/parse.py

```diff
@@ -1453,3 +1478,52 @@ def from_element(cls, element, api=None):
     def __str__(self):
         return '{self.name}@{self.version!r}'.format(self=self)
     __repr__ = __str__
+
+
+class SpecificationDocs(object):
```

Comment:
I think this fits better in the documentation module

Reply:
I based the `documentation,py` module on how the `specification.py` is structured: just classes that inherit from the base `Specification` class defined in `parse.py`. So `documentation.py` holds specialized classes that inherit from the base `SpecificationDocs` class, also defined in `parse.py`.